src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -2584,6 +2590,8 @@ private FrameworkElementFactory CreateFrozenButtonGrid()
 
             // Create frozen button
             var buttonFactory = new FrameworkElementFactory(typeof(Button));
+            buttonFactory.SetValue(System.Windows.Automation.AutomationProperties.NameProperty, "FrezzeButton");
+            buttonFactory.SetValue(System.Windows.Automation.AutomationProperties.AutomationIdProperty, "FrezzeButton");
```

Comment:
sorry missed this in the review, but there are spelling mistakes here we can still fix.
FYI @avidit @juliob-ad